CONTRIBUTING.md

```diff
@@ -348,62 +348,110 @@ Please use the template provided.
 
 ### Changelog
 
-If the PR you're about to open is a bugfix/feature visible to Biome users, you CAN add a new bullet point to [CHANGELOG.md](./CHANGELOG.md). Although **not required**, we appreciate the effort.
+This repository uses [knope](https://knope.tech/) to automate the releases of Biome's binaries, the Rust crates, the JavaScript libraries and the creation of the `CHANGELOG.md` for each library/crate.
 
-At the top of the file you will see a `Unreleased` section.
-The headings divide the sections by "scope"; you should be able to identify the scope that belongs to your change. If the change belongs to multiple scopes, you can copy the same sentence under those scopes.
+If the PR you're about to open is a bugfix/feature visible to Biome users, you are encouraged to provide a **changeset** . To *create* a changeset, use the following command (*don't create it manually*):
 
-Here's a sample of the headings:
+```shell
+just new-changeset
+```
+The command will present a prompt where you need to choose the libraries involved by the PR, the type of change (`major`, `minor` or `patch`) for each library, and a description of the change. The description will be used as name of the file.
 
-```markdown
-## Unreleased
+The command will create the changeset(s) in the `.changeset` folder. You're free to open the file, and add more information in it.
 
-### Analyzer
+#### Choose the correct packages
 
-### CLI
+In the vast majority of cases, you want to choose the `cli` library, which represent the main binary (AKA the npm package `@biomejs/biome`). If your PR also changes any *public* crate that is published, you'll have to select it too.
```

Comment:
It sounds a bit counterintuitive to call `cli` a library. Is that just Knope terminology here? Otherwise I’d suggest “component”.

Reply:
This name is arbitrary, but it will be used also in the git tags as prefix, and that's currently the nomenclature we use.

Changing it would mean to update `setup-biome` and other editor extensions to use a new nomenclature, which relies on those prefixes.

I opted for a name less meaningful, but less breaking changes to the bigger ecosystem

Reply:
Maybe we’re talking about something else? I didn’t mean it’s weird that `cli` is a component, just that the word “library” doesn’t really apply to it.

Reply:
Knope calls them "package", but in our case that `cli` is a black sheep because it's linked to multiple packages.

I don't mind using the word component